Codeinwp/neve
# feat: added upsell for hooks

### Summary

- Added Hooks upsell to Neve PRO
- Two strings are being added `Show Hooks` and `Hide Hooks` these are required as they are only available in PRO.

A proposed solution is to just use `Show` and `Hide` translations that we already have and use the `Hooks` string as an untranslated string. Eg. `__( 'Show', 'neve' ) . ' Hooks'` or `__( 'Hide', 'neve' ) . ' Hooks'`

### Will affect the visual aspect of the product

YES

### Screenshots 
![image](https://user-images.githubusercontent.com/23024731/223382587-747e0ced-7697-410d-91da-a235c1b9d119.png)

![image](https://user-images.githubusercontent.com/23024731/223382729-c403a562-e8ad-4d86-bebe-2de041779294.png)


### Test instructions

1. Check that if Neve Pro is not active the `Show Hooks` will be displayed on the front-end. 
2. Check that Upsell redirects to Neve Pro Upgrade page on Themeisle
3. Check that when Neve Pro is active it will use the functionality and hooks from Pro as before.


Closes Codeinwp/neve-pro-addon#2310.
